src/app/devices/deviceContent/reducer.ts

```diff
@@ -157,6 +158,28 @@ const reducer = reducerWithInitialState<DeviceContentStateType>(deviceContentSta
     })
     //#endregion
     //#region DigitalTwin-related actions
+    .case(getDigitalTwinAction.started, (state: DeviceContentStateType) => {
+        return state.merge({
+            digitalTwin: {
+                synchronizationStatus: SynchronizationStatus.working
+            }
+        });
+    })
+    .case(getDigitalTwinAction.done, (state: DeviceContentStateType, payload: {params: string} & {result: object}) => {
```

Comment:
is `{params: string} & {result: object}` different from `{params: string, result: object}`?

Reply:
syntax wise these two seems to be the same. I thought I c&p something different this time, but seems like the whole repo is following this pattern. If we want to change it, I can create a seperate PR to change them all.